queue_services/entity-digital-credentials/src/entity_digital_credentials/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.143.0'  # pylint: disable=invalid-name
```

Comment:
There were some changes in this code. For details, see https://github.com/bcgov/lear/releases.